src/pantry.js

```diff
@@ -12,6 +13,23 @@ class Pantry {
       }
     });
   }
+
+  compareIngredients(first, second) {
+    return first.ingredient === second.ingredient ? true : false;
+  }
+
+  checkPantryForRecipeIngredients = (recipe) => {
+    let supplyList = [];
+    
+    for (let i = 0; i < recipe.ingredients.length; i++) {
+      this.ingredients.forEach(ingredient => {
+        if (this.compareIngredients(recipe.ingredients[i], ingredient)) {
+          supplyList.push(ingredient);
+        }
+      }); 
+    }
```

Comment:
Could definitely refactor this - the forEach inside of the for loop seems redundant.